### ISSUE TITLE
custom-config.sh verifies F5 toolchain with /declare but should be /info as DO fails

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-google-gdm-templates#52]
## Description
The onboard logic has a shell file that is created called custom-config.sh. It deploys the tmsh commands and the AS3 app if found. Part of the logic in this script is to check the status of the F5 toolchain first. It does this in a function call and retrieves /info as well as /declare. However, /declare should not be checked as there should be nothing in /declare during a fresh boot box. Also, /declare is not an accurate check as it does not apply to all F5 toolchains like DO. DO does not have a /declare endpoint, and using the same function with DO "declarative-onboarding" will fail.

Current snippet...hitting /info and /declare
```
function wait_for_ready {
  app=$1
  checks=0
  ready_response=""
  ready_response_declare=""
  checks_max=10
  while [ $checks -lt $checks_max ] ; do
    ready_response=$(curl -sku admin:$passwd -w "%%{http_code}" -X GET  https://localhost:$${mgmtGuiPort}/mgmt/shared/$${app}/info -o /dev/null)
    ready_response_declare=$(curl -sku admin:$passwd -w "%%{http_code}" -X GET  https://localhost:$${mgmtGuiPort}/mgmt/shared/$${app}/declare -o /dev/null)
    if [[ $ready_response == *200 && $ready_response_declare == *204 ]]; then
        echo "$${app} is ready"
        break
```

Proposed snippet...hit only /info
```
function wait_for_ready {
  app=$1
  checks=0
  ready_response=""
  ready_response_declare=""
  checks_max=10
  while [ $checks -lt $checks_max ] ; do
    ready_response=$(curl -sku admin:$passwd -w "%%{http_code}" -X GET  https://localhost:$${mgmtGuiPort}/mgmt/shared/$${app}/info -o /dev/null)
    if [[ $ready_response == *200 ]]; then
        echo "$${app} is ready"
        break
```

This will also make the function more consistent since each F5 toolchain does indeed have /info as an endpoint. Not all toolchains have /declare as endpoint.

Refer to https://github.com/F5Networks/f5-declarative-onboarding/issues/148

## Template
All GDM templates

## Severity Level
3

**Comment by shyawnkarim** [comment]
Nice catch. I've filed a bug for this, internal ID ESECLDTPLT-2232, to fix this.

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-google-gdm-templates-v2).
